pkg/detectors/github/github.go

```diff
@@ -48,13 +56,49 @@ type userRes struct {
 // Keywords are used for efficiently pre-filtering chunks.
 // Use identifiers in the secret preferably, or the provider name.
 func (s Scanner) Keywords() []string {
-	return []string{"ghp_", "gho_", "ghu_", "ghs_", "ghr_", "github_pat_"}
+	return []string{"ghp_", "gho_", "ghu_", "ghs_", "ghr_", "github_pat_", "github"}
 }
 
 // FromData will find and optionally verify GitHub secrets in a given set of bytes.
 func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (results []detectors.Result, err error) {
 	dataStr := string(data)
 
+	// Oauth2 client ID and secret
+	oauth2ClientIDMatches := oauth2ClientIDPat.FindAllStringSubmatch(dataStr, -1)
+	oauth2ClientSecretMatches := oauth2ClientSecretPat.FindAllStringSubmatch(dataStr, -1)
+
+	for _, idMatch := range oauth2ClientIDMatches {
+		if len(idMatch) != 2 {
+			continue
+		}
+		for _, secretMatch := range oauth2ClientSecretMatches {
+			if len(secretMatch) != 2 {
+				continue
+			}
+
+			s1 := detectors.Result{
+				DetectorType: detectorspb.DetectorType_Github,
+				Raw:          []byte(idMatch[1]),
+			}
```

Comment:
If my understanding is correct, we should also have a `RawV2: []byte(idMatch[i] + secretMatch[1])`.

Reply:
https://github.com/trufflesecurity/trufflehog/pull/1584#discussion_r1281234786